robot-custom-integration/sl_python_robot/SLListener.py

```diff
@@ -28,11 +28,14 @@ def __init__(self, base_url, sltoken, bsid, stagename, labid=""):
             self.labid = bsid
 
     def start_suite(self, suite, result):
+        if not suite.tests:
+            return
+        print(f'{SEALIGHTS_LOG_TAG} {len(suite.tests)} tests in suite {suite.longname}')
         """initialize the test session so that all the tests can be identified by SeaLights as being part of the same session"""
         self.create_test_session()
         """request the list of tests to be executed from SeaLights"""
         self.excluded_tests = set(self.get_excluded_tests())
-        print(f'>>>> Skipped tests: {list(self.excluded_tests)}')
+        print(f'{SEALIGHTS_LOG_TAG} {len(self.excluded_tests)} Skipped tests: {list(self.excluded_tests)}')
```

Comment:
word "Skipped" should not be capitalized as it's not the first one in the sentence anymore.